peers/app_request_network.go

```diff
@@ -325,22 +330,31 @@ func (n *appRequestNetwork) TrackSubnet(subnetID ids.ID) {
 }
 
 func (n *appRequestNetwork) startUpdateValidators() {
+	n.wg.Add(1)
```

Comment:
a nit unrelated to this PR... If the entire function is wrapped in `go func() {}`, it would probably be easier just to call it as `go arNetwork.startUpdateValidators()` and not have the extra level of indentation